encoders/firebase-encoders/src/main/java/com/google/firebase/encoders/json/JsonValueObjectEncoderContext.java

```diff
@@ -101,6 +109,13 @@ public JsonValueObjectEncoderContext add(int value) throws IOException, Encoding
     return this;
   }
 
+  @NonNull
+  @Override
+  public JsonValueObjectEncoderContext add(long value) throws IOException, EncodingException {
+    jsonWriter.value(value);
```

Comment:
I think we want long values to be encoded as strings always.

Reply:
I don't think we have to, see my PR comment above. In the future we may have to do it, but we can have explicit logic for only encoding values `> 2^52-1` as strings. wdyt?

Reply:
sgtm